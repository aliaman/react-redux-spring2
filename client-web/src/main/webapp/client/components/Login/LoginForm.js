import React from 'react';
import { Row, Col, FormGroup, Form, FormControl, ControlLabel, HelpBlock, Button } from 'react-bootstrap';
import { connect } from 'react-redux'
import { doLogin } from '../../redux/actions/login/login'
import { hashHistory } from 'react-router'
import PropTypes from 'prop-types'
import ReactSpinner from 'reactjs-spinner'

@connect((store) => {
    return {
        userObj: store.login.userObj,
        authenticated: store.login.authenticated,
        error: store.login.error,
        fetching: store.login.fetching
    }
})
export default class LoginForm extends React.Component {
    constructor(props) {
        super(props);
    }
    componentWillReceiveProps(nextProps){
        this.setState(
            {
                userObj: nextProps.userObj,
                error: nextProps.error,
                authenticated: nextProps.authenticated
            },
            () => {
                if(this.state.authenticated){
                    //TODO: Fwd to roles homepage instead of hardcoded link
                    hashHistory.push('/dash');
                }
            }
        );
    }
    login(event){
        this.props.dispatch(doLogin(this.state.userObj.email, this.state.userObj.password));
        event.preventDefault();
    }
    componentWillMount() {
        this.setState({
            userObj: this.props.userObj,
            error: this.props.error,
            authenticated: this.props.authenticated
        });
    }
    handleChange (event){
        let varVal = this.state.userObj;
        varVal[event.target.id] = event.target.value;
        this.setState({
            userObj: varVal
        });
    }
    render(){
        return (
            <div>
                 {(this.props.fetching)
                     ?this.waitOrLoad()
                     :this.renderForm()}
            </div>
        )
    }
    waitOrLoad(){
        const style = {
            marginTop: '180px',
        };
        return (<div style={style}>
            <ReactSpinner size={50} borderColor={"#f3f3f3"} borderTopColor={"#3498db"} />
        </div>);
    }
    renderForm() {
        return (
            <div>
                <div className="error">{this.state.error}</div>
                <form onSubmit={this.login.bind(this)}>
                    <FormGroup>
                        <ControlLabel>Email address</ControlLabel>
                        <FormControl id="email"
                                     type="text"
                                     placeholder="Enter email"
                                     value={this.state.userObj.email}
                                     onChange={this.handleChange.bind(this)}/>
                    </FormGroup>
                    <FormGroup controlId="password">
                        <ControlLabel>Password</ControlLabel>
                        <FormControl id="password"
                                     type="password"
                                     value={this.state.userObj.password}
                                     onChange={this.handleChange.bind(this)}/>
                    </FormGroup>
                    <Button type="submit" >
                        Submit
                    </Button>
                </form>
            </div>
        );
    }
    _handleValidSubmit(values) {
        // Values is an object containing all values
        // from the inputs
    }

    _handleInvalidSubmit(errors, values) {
        // Errors is an array containing input names
        // that failed to validate
    }
}
LoginForm.propTypes = {
    error: PropTypes.string,
    userObj: PropTypes.object,
    authenticated: PropTypes.bool
}
LoginForm.defaultProps = {
    error:  null,
    authenticated: false,
    userObj: null
};