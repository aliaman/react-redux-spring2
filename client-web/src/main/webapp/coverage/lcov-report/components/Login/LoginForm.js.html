<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for components/Login/LoginForm.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../prettify.css" />
    <link rel="stylesheet" href="../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../index.html">All files</a> / <a href="index.html">components/Login</a> LoginForm.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">11.76% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>2/17</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/4</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>0/12</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">11.76% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>2/17</span>
      </div>
    </div>
  </div>
  <div class='status-line low'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49
50
51
52
53
54
55
56
57
58
59
60
61
62
63
64
65
66
67
68
69
70
71
72
73
74
75
76
77
78
79
80
81
82
83
84
85
86
87
88
89
90
91
92
93
94
95
96
97
98
99
100
101
102
103
104
105
106
107
108
109
110
111
112
113
114
115
116
117</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React from 'react';
import { Row, Col, FormGroup, Form, FormControl, ControlLabel, HelpBlock, Button } from 'react-bootstrap';
import { connect } from 'react-redux'
import { doLogin } from '../../redux/actions/login/login'
import { hashHistory } from 'react-router'
import PropTypes from 'prop-types'
import ReactSpinner from 'reactjs-spinner'
&nbsp;
@connect(<span class="fstat-no" title="function not covered" >(s</span>tore) =&gt; {
<span class="cstat-no" title="statement not covered" >    return {</span>
        userObj: store.login.userObj,
        authenticated: store.login.authenticated,
        error: store.login.error,
        fetching: store.login.fetching
    }
})
export default class LoginForm extends React.Component {
<span class="fstat-no" title="function not covered" >    co</span>nstructor(props) {
<span class="cstat-no" title="statement not covered" >        super(props);</span>
    }
<span class="fstat-no" title="function not covered" >    co</span>mponentWillReceiveProps(nextProps){
<span class="cstat-no" title="statement not covered" >        this.setState(</span>
            {
                userObj: nextProps.userObj,
                error: nextProps.error,
                authenticated: nextProps.authenticated
            },
<span class="fstat-no" title="function not covered" >            ()</span> =&gt; {
<span class="cstat-no" title="statement not covered" >                if(this.state.authenticated){</span>
                    //TODO: Fwd to roles homepage instead of hardcoded link
<span class="cstat-no" title="statement not covered" >                    hashHistory.push('/dash');</span>
                }
            }
        );
    }
<span class="fstat-no" title="function not covered" >    lo</span>gin(event){
<span class="cstat-no" title="statement not covered" >        this.props.dispatch(doLogin(this.state.userObj.email, this.state.userObj.password));</span>
<span class="cstat-no" title="statement not covered" >        event.preventDefault();</span>
    }
<span class="fstat-no" title="function not covered" >    co</span>mponentWillMount() {
<span class="cstat-no" title="statement not covered" >        this.setState({</span>
            userObj: this.props.userObj,
            error: this.props.error,
            authenticated: this.props.authenticated
        });
    }
<span class="fstat-no" title="function not covered" >    ha</span>ndleChange (event){
        let varVal = <span class="cstat-no" title="statement not covered" >this.state.userObj;</span>
<span class="cstat-no" title="statement not covered" >        varVal[event.target.id] = event.target.value;</span>
<span class="cstat-no" title="statement not covered" >        this.setState({</span>
            userObj: varVal
        });
    }
<span class="fstat-no" title="function not covered" >    re</span>nder(){
<span class="cstat-no" title="statement not covered" >        return (</span>
            &lt;div&gt;
                 {(this.props.fetching)
                     ?this.waitOrLoad()
                     :this.renderForm()}
            &lt;/div&gt;
        )
    }
<span class="fstat-no" title="function not covered" >    wa</span>itOrLoad(){
        const style = <span class="cstat-no" title="statement not covered" >{</span>
            marginTop: '180px',
        };
<span class="cstat-no" title="statement not covered" >        return (&lt;div style={style}&gt;</span>
            &lt;ReactSpinner size={50} borderColor={"#f3f3f3"} borderTopColor={"#3498db"} /&gt;
        &lt;/div&gt;);
    }
<span class="fstat-no" title="function not covered" >    re</span>nderForm() {
<span class="cstat-no" title="statement not covered" >        return (</span>
            &lt;div&gt;
                &lt;div className="error"&gt;{this.state.error}&lt;/div&gt;
                &lt;form onSubmit={this.login.bind(this)}&gt;
                    &lt;FormGroup&gt;
                        &lt;ControlLabel&gt;Email address&lt;/ControlLabel&gt;
                        &lt;FormControl id="email"
                                     type="text"
                                     placeholder="Enter email"
                                     value={this.state.userObj.email}
                                     onChange={this.handleChange.bind(this)}/&gt;
                    &lt;/FormGroup&gt;
                    &lt;FormGroup controlId="password"&gt;
                        &lt;ControlLabel&gt;Password&lt;/ControlLabel&gt;
                        &lt;FormControl id="password"
                                     type="password"
                                     value={this.state.userObj.password}
                                     onChange={this.handleChange.bind(this)}/&gt;
                    &lt;/FormGroup&gt;
                    &lt;Button type="submit" &gt;
                        Submit
                    &lt;/Button&gt;
                &lt;/form&gt;
            &lt;/div&gt;
        );
    }
<span class="fstat-no" title="function not covered" >    _h</span>andleValidSubmit(values) {
        // Values is an object containing all values
        // from the inputs
    }
&nbsp;
<span class="fstat-no" title="function not covered" >    _h</span>andleInvalidSubmit(errors, values) {
        // Errors is an array containing input names
        // that failed to validate
    }
}
LoginForm.propTypes = {
    error: PropTypes.string,
    userObj: PropTypes.object,
    authenticated: PropTypes.bool
}
LoginForm.defaultProps = {
    error:  null,
    authenticated: false,
    userObj: null
};</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Thu Jul 20 2017 13:06:18 GMT-0700 (PDT)
</div>
</div>
<script src="../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../sorter.js"></script>
</body>
</html>
