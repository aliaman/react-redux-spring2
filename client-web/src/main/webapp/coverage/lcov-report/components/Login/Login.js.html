<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for components/Login/Login.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../prettify.css" />
    <link rel="stylesheet" href="../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../index.html">All files</a> / <a href="index.html">components/Login</a> Login.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>3/3</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>2/2</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>3/3</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React, { Component, PropTypes } from 'react';
import LoginForm from './LoginForm';
import { Row, Col, FormGroup, Form, FormControl, ControlLabel, HelpBlock, Button } from 'react-bootstrap';
&nbsp;
export default class Login extends React.Component {
    constructor(props) {
        super(props);
    }
    render() {
        const rowStyles = {
            paddingTop: '100px'
        };
        return (
            &lt;div className="container"&gt;
                &lt;Row style={rowStyles}&gt;
                    &lt;Col lg={6} lgOffset={3}&gt;
                        &lt;h1 className="symheading"&gt;Login&lt;/h1&gt;
                        &lt;LoginForm/&gt;
                    &lt;/Col&gt;
                &lt;/Row&gt;
            &lt;/div&gt;
        );
&nbsp;
    }
}</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Thu Jul 20 2017 13:06:18 GMT-0700 (PDT)
</div>
</div>
<script src="../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../sorter.js"></script>
</body>
</html>
