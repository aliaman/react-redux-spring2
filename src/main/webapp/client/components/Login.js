import React, { Component, PropTypes } from 'react';
import LoginForm from './../scomponents/forms/LoginForm';

export default class Login extends React.Component {
    constructor(props) {
        super(props);
    }
    render(){
        return(
            <LoginForm />
        )
    }
}