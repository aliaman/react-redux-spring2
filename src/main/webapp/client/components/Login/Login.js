import React, { Component, PropTypes } from 'react';
import LoginForm from './LoginForm';
import { Row, Col, FormGroup, Form, FormControl, ControlLabel, HelpBlock, Button } from 'react-bootstrap';

export default class Login extends React.Component {
    constructor(props) {
        super(props);
    }
    render() {
        const rowStyles = {
            paddingTop: '100px'
        };
        return (
            <div className="container">
                <Row style={rowStyles}>
                    <Col lg={6} lgOffset={3}>
                        <h1 className="symheading">Login</h1>
                        <LoginForm/>
                    </Col>
                </Row>
            </div>
        );

    }
}