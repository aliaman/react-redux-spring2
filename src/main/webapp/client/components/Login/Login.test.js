import React from 'react'
import { shallow } from 'enzyme'
import Login from './Login'

test('Login component should render as expected', () => {
    const component = shallow(<Login />);
    expect(component.contains("Login")).toBe(true);
});