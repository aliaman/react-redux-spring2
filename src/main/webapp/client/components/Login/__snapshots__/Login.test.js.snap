// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component should render as expected 1`] = `
<div
  className="container"
>
  <Row
    bsClass="row"
    componentClass="div"
    style={
      Object {
        "paddingTop": "100px",
      }
    }
  >
    <Col
      bsClass="col"
      componentClass="div"
      lg={6}
      lgOffset={3}
    >
      <h1
        className="symheading"
      >
        Login
      </h1>
      <Connect(LoginForm)
        authenticated={false}
        error={null}
        userObj={null}
      />
    </Col>
  </Row>
</div>
`;
